docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,240 @@
+--
+title: Running ASP.NET MVC applications in Windows Docker Containers
+description: Learn how to take an existing ASP.NET MVC application and run it in a Windows Docker Container
+keywords: .NET, .NET Server, .NET Desktop
+author: BillWagner
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-framework-4.6
+ms.technology: dotnet-mvc
+ms.devlang: dotnet
+ms.assetid: c9f1d52c-b4bd-4b5d-b7f9-8f9ceaf778c4
+---
+
+# Running ASP.NET MVC Applications in Windows Docker Containers
+
+Running an existing .NET Framework based application in a Windows Docker container
+requires creating the Docker image that contains your application, and
+starting one or more containers to run that image. This topic explains
+the tasks you must perform to take an existing ASP.NET MVC application
+and deploy it in a Windows Docker container.
+
+You'll start with an existing ASP.NET MVC application. You'll see how to
+build the assets to publish using Visual Studio. You'll use the Docker tools
+to build the image that contains your application, and
+runs that application when it is started. You'll see how to connect to your
+application running in a Windows based Docker container.
+
+The application you'll run in a container is a simple website that
+answers questions randomly. This application
+is a basic MVC application with no authentication support, or database
+storage. That lets you focus on moving the web tier to a container. Later
+topics show how to move and manage persistent storage in containerized
+applications.
+
+The finished application is located in the
+[dotnet/core-docs repository on GitHub](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator).
+
+You need to be familiar with some Docker terms before you begin working
+on moving your application to a container.
+
+> A **Docker image** is a read-only template that defines the environment
```

Comment:
Switched to italic.
/cc @spboyer Do you want to include these definitions in your topic, and I'll remove them here.